winui/Scheduler/Day-Week-Views.md

```diff
@@ -516,5 +516,7 @@ this.Schedule.DaysViewSettings.CellRightPadding = 30;
 {% endhighlight %}
 {% endtabs %}
 
+![cell-right-padding-support-in-winui-scheduler-timeslot-views](DayandWeekViews_Images/cell-padding-in-winui-scheduler.png)
```

Comment:
Change image name to `cell-right-padding-support-in-WPF-scheduler`

change in all places.

Reply:
Changed.